src/coreclr/nativeaot/Runtime/CMakeLists.txt

```diff
@@ -1,9 +1,11 @@
 set(GC_DIR ../../gc)
+set(SHARED_RUNTIME_DIR ../../shared_runtime)
```

Comment:
A nit - we don't have any folders that have names with underscores, it would be nice to stay consistent.

Reply:
I'd like to hear opinions on the directory structure from several people here. @AaronRobinsonMSFT might also be interested to comment on this.

Reply:
`jit` and `gc` are shared too, but they do not have `shared_` in the name. I am not a fan of the `shared_` prefix.

Maybe just call it `runtime`?

Reply:
`runtime` seems appropriate. I agree that a prefix would be nice, but I agree with @janvorli's consistency argument. @jkotas's dislike of `shared_` is something I share too. I think `shared` is also dangerous as it breeds a dumping ground akin to `utils`.

Reply:
runtime it is. I'll make the changes soon.

Reply:
I'll be leaving the CMAKE name of the directory as SHARED_RUNTIME_DIR though, as RUNTIME_DIR is already taken for src/coreclr/nativeaot/Runtime

Reply:
Rename `RUNTIME_DIR` -> `NATIVEAOT_RUNTIME_DIR` and `SHARED_RUNTIME_DIR` -> `RUNTIME_DIR`? There are only 25 instances of `RUNTIME_DIR` spread over 4 files.

Reply:
Done.